File: RoRServer/src/main/java/commands/game/ChangeMapNameCommand.java
package commands.game;

import commands.base.CommandBase;
import communication.MessageInformation;
import models.session.GameSession;
import models.session.RoRSession;

/**
 * Setzt den neuen MapName der GameSession
 */
public class ChangeMapNameCommand extends CommandBase{

	private String mapName;
	
	public ChangeMapNameCommand(RoRSession session, MessageInformation messageInfo) {
		super(session, messageInfo);
		this.mapName = messageInfo.getValueAsString("mapName");
	}

	@Override
	public void execute() {
		GameSession gameSession = (GameSession)session;
		gameSession.setMapName(mapName);
	}

}

File: RoRClient/Commands/Game/Other/ChangeMapNameCommand.cs
using RoRClient.Commands.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using RoRClient.Communication.DataTransferObject;
using RoRClient.Models.Session;

namespace RoRClient.Commands.Game.Other
{
	class ChangeMapNameCommand : ChangeMapNameCommandBase
	{
		public ChangeMapNameCommand(RoRSession session, MessageInformation message) : base(session, message)
		{
		}
	}
}
